chart/templates/deployment.yaml

```diff
@@ -27,20 +28,67 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.deployment.args }}
           args:
+          {{- if .Values.deployment.args.defaultArgsEnabled }}
           {{- include "sealed-secrets-web.imageArgs" . | nindent 12 }}
+          {{- end }}
+          {{- if .Values.deployment.args.additionalArgs }}
+          {{- toYaml .Values.deployment.args.additionalArgs | nindent 12 }}
+          {{- end }}
+          {{- end }}
+          {{- if .Values.deployment.env }}
+          env:
+          {{- if .Values.deployment.env.SEALEDSECRETSCONTROLLERNAMESPACE }}
```

Comment:
use better readable camelCase  keys for the env variable in the values it only need to be uppercase in the variable name, not the helm values.